### ISSUE TITLE
Many things in Dashboard screen are not operational through a keyboard (Tab) interface.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#812]
**Describe the bug**
Many things in Dashboard screen are not operational through keyboard standard keys. If we want to access different Dashboard Gadgets on The Dashboard screen then we are not able to access many buttons as well as gadgets menu through Keyboard Keys such as Tab key.

1. Go to Dashboard screen(http://localhost:9992/cm/app/?view=dash&pathType=unknown)
2. See that all the options/controls on the screen are not accessible using standard Keyboard keys like Tab key.

**Expected behavior**
Whole page should be accessible using Standard keyboard Keys.

**Comment by rgarg24** [comment]
This issue got fixed by fixing other accessibility issues.
